src/gui/wallet.go

```diff
@@ -23,46 +23,44 @@ type SpendResult struct {
 	Error       string                     `json:"error,omitempty"`
 }
 
-// Spend spend coins from specific wallet
+// Spend spends coins from specific wallet
 func Spend(gateway *daemon.Gateway, walletID string, coins uint64, dest cipher.Address) *SpendResult {
```

Comment:
Return `(balance, transaction, error)` from this method
The caller of this should pack it into the `SpendResult` json struct